analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -216,6 +215,28 @@ public void Invocation_InstanceMethodCallDoesNotClearFieldForOtherAccess(string
             validator.ValidateTag("AfterStaticField", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
         }
 
+        [DataTestMethod]
+        [DataRow("StaticMethod();")]
+        [DataRow("Sample.StaticMethod();")]
+        public void Invocation_StaticMethodCallDoesNotClearFieldForOtherAccess(string invocation)
```

Comment:
I don't clearly understand the name. Does it mean this?
```suggestion
        public void Invocation_StaticMethodCall_DoesNotClearInstanceFields(string invocation)
```